code/modules/surgery/generic.dm

```diff
@@ -187,7 +187,7 @@
 	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 		if(..())
 			var/obj/item/organ/external/affected = target.get_organ(target_zone)
-			return affected && affected.open && target_zone != "mouth"
+			return affected && affected.open && affected.open <= 2.5 && target_zone != "mouth"
```

Comment:
this prevents you from cauterising an open ribcage. 
open = 3, ribcage is spread open
open = 2.5, ribcage is cut, but not pried open
open = 2 ribcage is intact but there's an open incisision

On most other servers, you would use bone gel to decrease it from 2.5 to 2 by repairing the cut ribcage. We don't have that item though